### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in setup.py

### ISSUE THREAD

**Issue by irfanpule** [ui/django-post_office#264]
hi, I have an error when running `pip install django-post-office` on my VM, but the command runs on my local machine.

```
ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-8pb7odoi/django-post-office/setup.py", line 40, in <module>
        long_description=open('README.rst').read(),
      File "/var/www/html/project/env/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 12118: ordinal not in range(128)
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-8pb7odoi/django-post-office/
```

**Comment by irfanpule** [comment]
I tried installing it manually by adding explicit coding on line 41 `long_description = open ('README.rst', encoding ="utf-8"). Read () `

> https://github.com/ui/django-post_office/blob/master/setup.py#L41

it works on my VM and local machine, but I don't understand where the core problem.

**Comment by jrief** [comment]
Just released version 3.2.1. Please read the Changelog for details. In 3.2.1 it should be fixed. Using the `encoding="utf-8"` causes other problems.
